test/e2e/capo.go

```diff
@@ -75,9 +74,6 @@ func CreateClusterAndWait(ctx context.Context, input CreateClusterAndWaitInput,
 	Expect(input.ConfigCluster.ControlPlaneMachineCount).ToNot(BeNil())
 	Expect(input.ConfigCluster.WorkerMachineCount).ToNot(BeNil())
 
-	log.Logf("Creating the workload cluster with name %q using the %q template (Kubernetes %s, %d control-plane machines, %d worker machines)",
```

Comment:
is this related to this PR (upgrade ? )
Why removing this ?

Reply:
Yes, "k8s.io/kubernetes/test/e2e/framework/log" is deprecated